packages/@aws-cdk/cdk-deploy/package.json

```diff
@@ -0,0 +1,82 @@
+{
+  "name": "@aws-cdk/cdk-deploy",
```

Comment:
This module should be `cdk-deploy` not `@aws-cdk/cdk-deploy`, so users can do `npx cdk-deploy`. As a general rule I recon, we don't put tools under `@aws-cdk`